Watchman.AwsResources/Services/Sqs/QueueSource.cs

```diff
@@ -76,7 +76,7 @@ protected override async Task<IEnumerable<QueueData>> FetchResources()
         {
             var names = await ReadActiveQueueNames();
 
-            return names.Select(n => new QueueData() {Name = n}).ToList();
+            return names.Select(n => new QueueData() {Name = n, IsErrorQueue = n.EndsWith("_error")}).ToList();
```

Comment:
the error suffix is configurable on the old interface (https://github.com/justeat/AwsWatchman/blob/1cd80b44135569b0623d3891f3214ec667b936a6/ConfigurationFileFormat.md#errors) , so was thinking we should probably preserve the ability to change it? might be also good to make it a case-insensitve check?